commands/heroku/heroku-ps.md

```diff
@@ -0,0 +1,23 @@
+### List running dynos
+
+`heroku ps {{TYPE}} --app {{APP_NAME}}`
+
+- <b>TYPE</b>: Type of dyno.  This is optional. If it is not given then all the dynos are listed.
+- <b>APP_NAME</b>: Name of the app.
+
+#### Example:
+
+`heroku ps --app TestApp`
+
+=== run: one-off dyno
```

Comment:
What does this means? As in View mode it looks something like this

<img width="536" alt="screen shot 2019-01-01 at 8 07 14 pm" src="https://user-images.githubusercontent.com/4654382/50573813-e2d66100-0e00-11e9-84bb-2e92c16c650e.png">

Reply:
is this suppose to be in single line?

Reply:
I have moved the run.1 & web.1 to next line

run & web denotes the type of dyno and then the command which created the dyno.
Next line shows the dyno number along with its up time.

Reply:
So it's basically the response of the command right?
At present we are not planning to put commands output here.
I was thinking of adding a gif of command and output of the command. But haven't started working on it.
If you want you can add the gif, of what happens on executing commands or just the command will be fine.

If you have any inputs on this, I would love to hear it.